src/server/game/World/World.cpp

```diff
@@ -3393,3 +3401,24 @@ CliCommandHolder::~CliCommandHolder()
 {
     free(m_command);
 }
+
+void World::AddDelayedDamage(Unit * attacker, Unit * victim, uint32 damage, CleanDamage const* cleanDamage, DamageEffectType damagetype, SpellSchoolMask damageSchoolMask, SpellInfo const* spellProto, bool durabilityLoss)
```

Comment:
\*

Reply:
Will make the adjustment and test before pushing.

Reply:
https://github.com/azerothcore/azerothcore-wotlk/pull/14199/commits/bc3ea4d68565d7668c82e93aa538db6c36c770c6